script.js

```diff
@@ -23,17 +23,32 @@ const addTodoHandler = () => {
     renderTodos(todos);
 };
 
+const deleteTodoHandler = (todoId) => {
+    todos = [...todos].filter((todo) => todo.id !== todoId);
```

Comment:
No need to spread as spreading is an `O(n)` operation. Also there is no issues or reference.
```suggestion
    todos = todos.filter((todo) => todo.id !== todoId);
```